File: assignments/solutions/02-first-java/greetingUser.js
/**
 * Greeting User
 * 
 * @param {String} input
 * @returns {String}
 */

const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

console.log('Greeting User');
rl.question('Enter your name: ', input => {
    console.log(`Hello ${input}!`);
    rl.close();
});

/**
 * Output:
 * 
 * Greeting User
 * Enter your name: Saideep
 * Hello Saideep!
 */

File: assignments/solutions/02-first-java/greetingUser.java
// Greeting User

import java.util.Scanner;

public class greetingUser {
    public static void main (String args[]) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Greeting User\n");
        System.out.print("Enter your name: ");
        String input = scan.nextLine();
        System.out.println("Hello, " + input + "!");
    }
}

/*
Output;

Greeting User

Enter your name: Saideep
Hello, Saideep!
*/